tests/libtest.sh

```diff
@@ -21,8 +21,8 @@ lv_exists() {
 }
 
 clean_config_files() {
-  rm -f /etc/sysconfig/docker-storage-setup
-  rm -f /etc/sysconfig/docker-storage
+  rm -f $1
+  rm -f $2
```

Comment:
We had got rid of clean_config_file() function. It has come back. what's going on.